conf/vertebrates/mlss_conf.xml

```diff
@@ -90,6 +94,7 @@
     </one_vs_all>
 
     <!-- Early vertebrates and other chordates -->
+    <one_vs_all method="LASTZ_NET" ref_genome="xenopus_tropicalis" against="Amphibia"/>
     <one_vs_all method="LASTZ_NET" ref_genome="ciona_intestinalis" against="Ciona" ref_amongst="Cyclostomata"/>
```

Comment:
Is this alignment even necessary? _Ciona_ is an invertebrate, so isn't used in any EPOs. This has probably always been here, because we use _Ciona_ as an outgroup to vertebrates. But what do we use the alignment for? This feels more like a Pan thing if we did more than just Pan homology. Just a comment.

Reply:
The line between the www and metazoa sites is indeed blurry. Cionas were initially put on the www site, and I think EG didn't exist at the time. From there, all chordates are meant to be on the www site, even though in reality we have a much stronger focus on Vertebrates.
I seem to remember that amphioxus was somehow still to go to Metazoa, but I can't see it on any site, so maybe it has been delayed

Reply:
I think we should have all the chordates then! :sweat_smile: